src/nbl/video/COpenGLShader.cpp

```diff
@@ -11,36 +11,36 @@ using namespace nbl::video;
 // TODO: deduce the following in COpenGLPhysicalDevice from the OpenGL extension presence, and then add the defines as standardy from IPhysicalDevice : addCommonGLSLDefines() instead
 // The subgroup stuff is a bit hard cause we made up "extra" extensions such as `NBL_GL_KHR_shader_subgroup_ballot_inverse_ballot_bit_count` (probably best to expose as extra features)
 const char* COpenGLShader::k_openGL2VulkanExtensionMap = R"(
```

Comment:
remind me to clean this shit up and redo the prefix sum sample/workgroup ops example